src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -27,6 +27,11 @@ const CLP_OPTION_MAP = Dict(
     :Algorithm => (Clp_algorithm, Clp_setAlgorithm)
 )
 
+# These two setters have a third argument `extraInfo`, so we extend them
+# to avoid errors when setting the corresponding `MOI.RawAttribute`s
+ClpSolve_setPresolveType(arg1, amount) = ClpSolve_setPresolveType(arg1, amount, -1)
+ClpSolve_setSolveType(arg1, method) = ClpSolve_setSolveType(arg1, method, -1)
```

Comment:
I am very strongly against adding methods like this.

We should fix the handling of the `RawParameter` to account for the need to pass multiple arguments.

```Julia
function MOI.set(model, p::MOI.RawParameter, args...)
    # ... some stuff ...
    ClpSolve_setSolveType(model.inner, args...)
end

Reply:
Or we do
```Julia
function MOI.set(model, p::MOI.RawParameter, value)
    if p.name == :SolveType
        ClpSolve_setSolveType(model.solver_options, value, -1)
    else
        error("Parameter $p not valid")
    end
end
```

Reply:
These `if` statements are probably nicer than the dictionaries